### PR TITLE
Feature/audi table

### PR DESCRIPTION
**Pull request by purushothamanTR**
pull request

### CODE REVIEW

File: BookingSystem.BAL/Service/BookingPartialClass/CancelBooking.cs [hunk 1]
```diff
@@ -9,7 +9,8 @@ public partial class BookingService : IBookingService
        public async Task<BookingCancelVM> CancelBookingAsync(BookingCancelVM booking)
         {
             await _attributeRepository.CancelBookingAsync(booking).ConfigureAwait(false);
-            await _refundRepository.SaveRefundTransactionAsync(booking).ConfigureAwait(false);
+            await _bookingRepository.UpdateCancelBookingAsync(booking).ConfigureAwait(false);
+            //await _refundRepository.SaveRefundTransactionAsync(booking).ConfigureAwait(false);
```
Comment (manic1990) [inline BookingSystem.BAL/Service/BookingPartialClass/CancelBooking.cs:13]: remove unwanted lines

File: BookingSystem.DAL/Data/DataContext.cs [hunk 2]
```diff
@@ -1,24 +1,34 @@
...
 
+
 namespace BookingSystem.DAL.Data
 {
-   public class DataContext :DbContext
+    public class DataContext : DbContext
     {
-        public DataContext(DbContextOptions<DataContext> options) : base(options) { }
+
+        private readonly IConfiguration _Configuration;
```
Comment (manic1990) [inline BookingSystem.DAL/Data/DataContext.cs:17]: remove empty spaces check places too.

File: BookingSystem.BAL/Service/BookingPartialClass/CancelBooking.cs (hunk 1 above)
Comment (purushothamanTR) [reply BookingSystem.BAL/Service/BookingPartialClass/CancelBooking.cs:13]: removed unwanted lines

File: BookingSystem.DAL/Data/DataContext.cs (hunk 2 above)
Comment (purushothamanTR) [reply BookingSystem.DAL/Data/DataContext.cs:17]: removed empty spaces

**Pull request merged**